tests/test_openapi_schema_publishers_id.py

```
@@ -0,0 +1,34 @@
+from typing import Any
+
+import pytest
+from ninja.openapi.schema import OpenAPISchema
+
+from tests.utils import IntegerKeyJSONDecoder
+
+
+@pytest.fixture
+def openapi_schema() -> OpenAPISchema:
+    """Get the OpenAPI schema."""
+    # return client.get("/api/openapi.json").json()
```

Ruff (0.8.2)

Found commented-out code

Remove commented-out code

(ERA001)